worlds/hk/docs/en_Hollow Knight.md

```diff
@@ -20,3 +20,10 @@ When the Hollow Knight player picks up an item from a location and it is an item
 player's recent items display as an item being sent to another player. If the item is for another Hollow Knight player
 then the sprite will be that of the item's original sprite. If the item belongs to a player that is not playing Hollow
 Knight then the sprite will be the Archipelago logo.
+
+## What logic does the randomization rely on?
+
+Randomization may expect you to use the Benchwarp mod to return to the starting location. Warping elsewhere will never be required to complete the game. For more details, see the documentation for the Hollow Knight randomizer mod, below:
+
+[README](https://github.com/homothetyhk/RandomizerMod/blob/master/README.md)
```

Comment:
While these are good resources, I'd prefer if we made more clear that these are resources for the standalone randomizer that may not be completely accurate to Archipelago.